bentoml/_internal/bento/bento.py

```diff
@@ -334,6 +340,21 @@ def save(
             fs.mirror.mirror(self._fs, out_fs, copy_if_newer=False)
             self._fs.close()
             self.__fs = out_fs
+            event_properties = {
+                "bento_tag": str(self.info.tag),
+                "bento_creation_timestamp": self.info.creation_time.isoformat(),
+                "bento_size": human_readable_size(calc_dir_size(bento_path)),
+                "models_tags": [str(i) for i in self.info.models],
+                "models_type": [
+                    model_store.get(i).info.module for i in self.info.models
+                ],
+            }
+
+        track(
+            BENTO_BUILD_TRACK_EVENT_TYPE,
+            event_pid=os.getpid(),
```

Comment:
why do we want to pid here?

Reply:
pid is a common field that we want to track?